vmdb/spec/controllers/catalog_controller_spec.rb

```diff
@@ -135,7 +130,7 @@
     it "Orchestration Template is copied" do
       controller.instance_variable_set(:@sb, {})
       controller.instance_variable_set(:@_params, :button => "save")
-      controller.instance_variable_set(:@_response, ActionController::TestResponse.new)
+      controller.should_receive(:render)
       ot = FactoryGirl.create(:orchestration_template)
       controller.x_node = "xx-ot_othot-#{ot.id}"
```

Comment:
this can be done on line 131 controller.instance_variable_set(:@sb, {:active_tree => "ot_tree", :ot_tree => "xx-ot_othot-#{ot.id}})